### PR TITLE
built Index page

### PR DESCRIPTION
**Pull request by krisrjohnson21**
- connected react front end with rails back end
- built app component with root route and index route
- built index component to render CapeTiles 
- built CapeTile component to render props for each superhero
- built and migrated Capes database
- wrote enzyme tests for presentational component
- wrote rspec tests for API controller

### CODE REVIEW

File: app/javascript/react/components/capes/CapeTile.test.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import React from "react"
+import Enzyme, { mount } from "enzyme"
+
+import Adapter from "enzyme-adapter-react-16"
+
+Enzyme.configure({ adapter: new Adapter() })
+
+import CapeTile from "./CapeTile"
+import { BrowserRouter } from "react-router-dom"
```
Comment (nwalberts) [inline app/javascript/react/components/capes/CapeTile.test.js]: I would import all of the "libraries" and config setup at the top, and the import of the cape tile slightly seperate

File: app/javascript/react/components/capes/CapeTile.test.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+          gender="Male"
+          affiliation="Lornch Ablademy"
+          intelligence={99}
+          strength={5}
+          speed={40}
+          url="https://cdn.bulbagarden.net/upload/thumb/a/a6/Roxanne_Nosepass_Adventures.png/200px-Roxanne_Nosepass_Adventures.png"
+        />
+      </BrowserRouter>
+
+    )
```
Comment (nwalberts) [inline app/javascript/react/components/capes/CapeTile.test.js]: remove extra space

File: app/javascript/react/components/capes/CapesList.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        id={cape.id}
+        name={cape.name}
+        full_name={cape.full_name}
+        gender={cape.gender}
+        affiliation={cape.affiliation}
+        intelligence={cape.intelligence}
+        strength={cape.strength}
+        speed={cape.speed}
+        url={cape.url}
+      />
```
Comment (nwalberts) [inline app/javascript/react/components/capes/CapesList.js]: this is a LOT of props to pass. Just pass one prop: cape=cape, and then unpack them in your CapeTile component

File: app/javascript/react/components/superhero/Superheroes.jsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+  return(
+    <>
+    <h1>Search results list</h1>
+    </>
+  )
+}
+
+
+export default Superheroes;
```
Comment (nwalberts) [inline app/javascript/react/components/superhero/Superheroes.jsx]: Nope. This is an unfinished feature, remove until you designate an actual feature story for this. We will not merge in unfinished feature code :/

File: app/javascript/react/components/capes/CapesList.js (hunk 3 above)
Comment (krisrjohnson21) [reply app/javascript/react/components/capes/CapesList.js]: Nick, we've been trying to pass this down for a couple hours and it's breaking all of our tests. Is it possible for us to continue passing down these props? We've tried refactoring our tests extensively with no luck, and Zain/Vinh advised they ran into the same problem and solved it by passing props down individually. Thanks in advance for your guidance!

File: app/controllers/api/v1/capes_controller.rb [hunk 5]
```diff
@@ -0,0 +1,6 @@
+class Api::V1::CapesController < ApiController
+
```
Comment (safang99) [inline app/controllers/api/v1/capes_controller.rb]: remove blank line

File: app/javascript/react/components/capes/CapeTile.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+      <h4>Affiliation: {affiliation}</h4>
+      <h5>Intelligence: {intelligence}</h5>
+      <h6>Strength: {strength}</h6>
+      <p>Speed: {speed}</p>
+      <img src={url} />
+    </div>
+  )
+}
+
+
```
Comment (safang99) [inline app/javascript/react/components/capes/CapeTile.js]: remove extra blank line.  There only needs to be one.

**Comment by krisrjohnson21** [comment]
-Removed blank line from CapeTile and blank line from api::v1::capes controller

**Pull request merged**